src/Oscoin/Protocol/Sync.hs

```diff
@@ -176,35 +201,35 @@ data SyncContext c tx s m = SyncContext
     -- variation of it.
     , scLocalChainReader  :: BlockStoreReader c tx s m
     -- ^ A read-only handle to the local blockchain.
-    , scUpstreamConsumers :: [SyncEvent c tx s -> m ()]
+    , scEventHandlers :: [SyncEvent c tx s -> m ()]
     -- ^ A list of \"upstream consumers\", represented as effectful actions from
```

Comment:
Comment needs updating :)

Reply:
Can do here or in my (final) PR (which I will hopefully submit by the end of the day) -- whatever works best for you 😉

Reply:
:man_shrugging: